src/components/button/button.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import {ButtonPrimary, ButtonSecondary, ButtonIcon, ButtonText} from "./button"
+
+export default {
+  title: "Button",
+  component: ButtonPrimary,
+};
+
+export const ButtonStory = () => {
+  return (
+    <> 
+    <ButtonPrimary /> 
```

Comment:
Once you make the requested change below, please also update this file so that there is an example of each. Make sure you pass in props to each one to showcase various use-cases. You can add click listeners too with functions like `() => alert('Hello World!')` to showcase the onClick functionality. Ideally, each button type would be a separate export by itself so that they show up as separate menu items on the left sidebar in storybook like so:

```
export const PrimaryButton = () { ... }

export const SecondaryButton = () { ... }

export const TextButton = () { ... }

export const IconButton = () { ... }
```